packages/coding-standard/docs/rules_overview.md

```diff
@@ -1,10 +1,10 @@
-# 14 Rules Overview
+# 15 Rules Overview
 
 ## ArrayListItemNewlineFixer
 
 Indexed PHP array item has to have one line per item
 
-- class: `Symplify\CodingStandard\Fixer\ArrayNotation\ArrayListItemNewlineFixer`
+- class: [`Symplify\CodingStandard\Fixer\ArrayNotation\ArrayListItemNewlineFixer`](packages/coding-standard/src/Fixer/ArrayNotation/ArrayListItemNewlineFixer.php)
```

Comment:
The link seems invalid, need to have `/` prefix to avoid read on current directory.